vcenter_operator/configurator.py

```diff
@@ -255,12 +237,6 @@ def _poll(self, host):
                         # we try to query its name here
                         continue
 
-                if 'bridge' not in cluster_options and not nsx_t_enabled:
```

Comment:
Where does this `bridge` come from and do we still need it? Looks like it was only used in the old dvs-agent's configmap: https://github.com/sapcc/helm-charts/blob/8ac84a95a4c9572a425118a9bbfa5c1953a167a5/openstack/nova/templates/hypervisors/hypervisors-vmware-neutron-configmap-vct.yaml#L39
Maybe we can remove both?

Reply:
Created a PR to drop the old neutron stuff in nova: https://github.com/sapcc/helm-charts/pull/4987